adapter/internal/messaging/notification_listener.go

```diff
@@ -164,6 +168,32 @@ func handleAPIEvents(data []byte, eventType string) {
 	}
 }
 
+func handleLifeCycleEvents(data []byte) {
+	var apiEvent APIEvent
+	json.Unmarshal([]byte(string(data)), &apiEvent)
+	conf, _ := config.ReadConfigs()
+	configuredEnvs := conf.ControlPlane.EnvironmentLabels
+	logger.LoggerMsg.Debugf("%s : %s API life cycle state change event triggered", apiEvent.APIName, apiEvent.APIVersion)
+	if len(configuredEnvs) == 0 {
+		configuredEnvs = append(configuredEnvs, eh.DefaultGatewayLabelValue)
+	}
+	for _, configuredEnv := range configuredEnvs {
+		if _, ok := eh.APIListMap[configuredEnv]; ok {
+			apiListOfEnv := eh.APIListMap[configuredEnv].List
+			for i := range apiListOfEnv {
+				if apiEvent.UUID == apiListOfEnv[i].UUID && apiListOfEnv[i].APIStatus == blockedStatus ||
```

Comment:
shall we improve readability with paranthesies

Reply:
Fixed